test/test_nn.py

```diff
@@ -2606,6 +2607,61 @@ def test_orthogonal(self):
                                          torch.eye(rows) * gain ** 2, prec=1e-6)
 
 
+# functional loss wrappers
+
+
+class _SmoothL1LossFunctional(_Loss):
```

Comment:
I'm not sure this is the best way to test them. I actually just noticed that we don't have tests for any of the modules in `functional`, and are relying only on tests for their `Module` counterpart.
I'd say that as the `Module` call into `functional`, in a way that mimics `functional` interface with classes, it would be enough to test `Module` only. @apaszke wdyt?